debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (native)
```

Comment:
See comment above, this should not be a native package.

Reply:
Yes, it's a mistake. I'm using `3.0 (quilt)` in my other Debian packages.

Note that providing a `debian` directory does not harm anymore since it is ignored by `dpkg-source` when using the `3.0 (quilt)` format.